lib/pages/job_detail_page/job_detail_page.dart

```diff
@@ -161,11 +162,52 @@ class _JobDetailPageState extends State<JobDetailPage> {
                   ),
                   Container(
                     width: double.infinity,
-                    height: 150,
+                    height: 180,
                     child: Card(
                       margin: const EdgeInsets.all(10.0),
                       elevation: 5,
-                      child: Center(child: Text('Time Table')),
+                      // NOTE
+                      child: Padding(
+                        padding: const EdgeInsets.all(10.0),
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.spaceAround,
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Text('TimeTable',
+                                style: Theme.of(context).textTheme.headline6),
+                            ...jobDetail['working_hour'][0]['day_of_week']
+                                .map(
+                                  (day) => Row(
+                                    mainAxisAlignment:
+                                        MainAxisAlignment.spaceBetween,
+                                    children: [
+                                      Text(weeksName(weekEnum.values[day]),
+                                          style: Theme.of(context)
+                                              .textTheme
+                                              .bodyText1),
+                                      SizedBox(width: 10),
+                                      Row(
+                                        children: [
+                                          Text(
+                                            (jobDetail['working_hour'][0]
```

Comment:
this part can be rewritten, instead of using Row(), just use Text('<start time> - <end time>')
and better put working hour business logic in bloc.